sale_order_general_discount/models/sale_order_line.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    discount = fields.Float(
+        compute="_compute_general_discount",
```

Comment:
Use convention for name and for not having problems with other inheritances:
```suggestion
        compute="_compute_discount",
```